modules/vba_documents/spec/models/upload_file_spec.rb

```diff
@@ -21,6 +21,6 @@
   it 'does not instantiatiate on UploadSubmission' do
     upload_model = VBADocuments::UploadSubmission.new
     upload_model.save!
-    expect { VBADocuments::UploadFile.find_by(guid: upload_model.guid) }.to raise_error(ActiveRecord::StatementInvalid)
+    expect { VBADocuments::UploadFile.find_by(guid: upload_model.guid) }.to raise_error(TypeError)
```

Comment:
I cannot find the exact change, but in previous versions it seems that errors raised during activerecord execution were raised as `ActiveRecord::StatementInvalid`. Now it seems that it raises the error explicitly raised in the code